library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -150,21 +170,61 @@ class Client() {
         val clientOptions = options ?: ClientOptions()
         val apiClient =
             GRPCApiClient(environment = clientOptions.api.env, secure = clientOptions.api.isSecure)
-        return Client(address = address, privateKeyBundleV1 = bundle, apiClient = apiClient)
+        val v3Client: FfiXmtpClient? = runBlocking {
+            ffiXmtpClient(
+                options,
+                null,
```

Comment:
I don't think we can skip the signer here (see other comment)

Reply:
This case only happens with static keys. So you are passing in a bundle not a signer object. I guess I can make a privateKey from that bundle if it's required 🤔

Reply:
This param is intended to be specifically used for wallet signing, as opposed to signing with the legacy identity key. Even if a legacy identity key was passed in, and it was obtained from a cached/static key, libxmtp may sometimes still need a wallet signature. I'll write a doc later to make all of this clear!

Reply:
Oh - @nplasterer is the issue that the final app integrator won't give us a signer if they give us a cached bundle?

Reply:
Correct! Should I go back to the null implementation?